csfieldguide/chapters/content/en/artificial-intelligence/sections/chatterbots-and-the-turing-test.md

```diff
@@ -103,7 +102,7 @@ If you want to know more, just ask Alice how she works, and she’ll tell you!
 
 Some possible rules that Alice might have are:
 
-- If "favourite colours" are mentioned,  say your favourite colour is "green".
+- If "favourite colours" are mentioned, say your favourite colour is "green".
 - If family members are mentioned (e.g. "mum", "dad", or "brother") say something that is about family.
```

Comment:
Should we capitalise mum, dad, etc here?

Reply:
They aren't used specifically as pronouns here so I don't think so